onboarding/serializers.py

```diff
@@ -283,8 +283,17 @@ class Meta:
             'page',
         )
 
+    def validate(self, data):
+        '''
+        Prevents data in Section from duplicating from backend side (when data is
+        re-send manually; problem was previously solved in frontend for the user).
+        '''
+        if Section.objects.filter(title = data['title'],
+                description = data['description'], link = data['link'],
+                owner = self.context['request'].user.company).exists():
+            raise serializers.ValidationError("Dane już istnieją w bazie - duplikat")
+        return data
 
-# ANSWER
```

Comment:
It's pity you ignore introduce comments :/

Reply:
idk what "ANSWER" means :D

Reply:
Many serializers have a hint-comment above what is this serializer for,
meaning which DB model or what relation.
If it was set up at the time when this file was created
then why ruin this ;)